cfstore/interface.py

```diff
@@ -440,22 +440,57 @@ def delete_file_from_collection(self, collection, file):
         self.session.commit()
 
         
-    def retrieve_variable(self, **kw):
+    def retrieve_variable(self, key, value):
```

Comment:
There are two methods here with the same docstring. What is the difference? Can you please adjust the docstrings?